### ISSUE TITLE
Lens distortion

### ISSUE THREAD

**Issue by vincentgillanders** [SUDO-AI-3D/zero123plus#26]
Hi there,
Firstly, congratulations on this project, and thank you for all your efforts.
I have a question about the output.  A use case I'd like to explore is creating assets for a isometric game from predominantly front-on images. Z++ output seems to generate a fitting camera angle, but due to perspective lens distortion, objects appear irregular comparatively to isometric environments. 
_eg. sofa which appears different heights on each end due to objects appearing larger the closer they are to the camera._

Do you happen to foresee options for Zero123++ which would try to generate more isometric-friendly imagery in the future?

Thank you again.

**Comment by eliphatfs** [comment]
Yes; we mainly train Zero123++ with a synthetic dataset, and as we did not yet consider real camera parameters like lens distortion, we have found that the model is limited in adaptivity to these parameters that exist in the real world. We have a plan to train a better version of Zero123++ with things like extreme FOV (including orthographic projection/isometric view), lens distortions, chromatic aberration, etc. in mind.
